web-platform/functions/src/studies.ts

```diff
@@ -52,6 +52,7 @@ export const studies = {
     schemaNamespace: "rally-attention-stream",
     downloadLink: {
       chrome: "https://chrome.google.com/webstore/detail/attention-stream/bahhehaddofgkccippmjcecepdakppme",
+      firefox: "https://addons.mozilla.org/en-US/firefox/addon/attention-stream/",
```

Comment:
OK with the caveat that it's not guaranteed to be available when we are finally able to upload the extension (which is required to create the store page) :)